cpp/src/strings/convert/convert_datetime.cu

```diff
@@ -747,34 +753,18 @@ struct from_timestamp_base {
 
     return result;
   }
-};
 
-template <typename T>
-struct from_timestamps_size_fn : public from_timestamp_base<T> {
```

Comment:
Basically combining `from_timestamps_size_fn` and `datetime_formatter` functors into one functor: `datetime_formatter_fn`